code/controllers/configuration/entries/comms.dm

```diff
@@ -25,3 +25,9 @@
 
 /datum/config_entry/string/cross_comms_network
 	protection = CONFIG_ENTRY_LOCKED
+
+/datum/config_entry/string/hell
+	default = "byond:\\terry.tgstation13.org:3336"
+
+/datum/config_entry/string/the_abyss
+	default = "byond:\\campbell.tgstation13.org:6337"
```

Comment:
//

Reply:
I just followed the format used elsewhere for server addresses (see a few lines above this, line 22). I'll change it if you want tho